android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/PatientExtension.kt

```diff
@@ -36,7 +35,7 @@ private fun String.toTitleCase() = replaceFirstChar {
   if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString()
 }
 
-fun Patient.extractGender(context: Context) =
+fun Patient.extractGender() =
```

Comment:
Removing context would not allow for translation. Suggestion is to have the text in `string.xml` file then pick the first letter. Example **Female**  is translated to **Kvenkyns**  in Icelandic (courtesy of Google translate). The first letters are not the same.